src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -3144,21 +3144,29 @@ public void testExecInspect() throws Exception {
         containerId, new String[] {"sh", "-c", "exit 2"},
         createParams.toArray(new ExecCreateParam[createParams.size()]));
     final String execId = execCreation.id();
-
     log.info("execId = {}", execId);
+
+    final ExecState notStarted = sut.execInspect(execId);
+    assertThat(notStarted.id(), is(execId));
+    assertThat(notStarted.running(), is(false));
+    assertThat(notStarted.exitCode(), nullValue());
```

Comment:
Failed Travis builds on 1.8 and 1.9 indicate the exit code for non-started execs are 0 instead of null.